app-preprod.yaml

```diff
@@ -76,9 +79,9 @@ spec:
   env:
     - name: SPRING_PROFILES_ACTIVE
       value: preprod
-    - name: CREDENTIAL_USERNAME
-      value: /run/secrets/nais.io/srvfamilie-tilbake/username
-    - name: CREDENTIAL_PASSWORD
-      value: /run/secrets/nais.io/srvfamilie-tilbake/password
+    - name: CREDENTIAL_USERNAME_PATH
```

Comment:
Det kan ikke gjøres, tror jeg. Disse verdiene brukes vi for å authentisere iverksettelse

Reply:
Ligger allerede i vault(lagt inn manuellt), så egentlig bare flaks at disse ikke ødela :)